### ISSUE TITLE
TransactionID no longer used by Feeder Gateway, feeder.go still includes code to use it. [LOW SEVERITY]

### ISSUE THREAD

**Issue by D-DePablos** [NethermindEth/juno#211]
**Describe the bug**
See title

**To Reproduce**
GET request to: https://alpha-mainnet.starknet.io/feeder_gateway/get_transaction?transactionId=5425206

**Expected behavior**
Returns transaction information (in both Feeder gateway and feeder implementation)

**Screenshots**
<img width="1361" alt="image" src="https://user-images.githubusercontent.com/42889395/173066922-8ba6e936-3224-44c9-95ce-f534034b7392.png">

![image](https://user-images.githubusercontent.com/42889395/173067017-26ef37d4-f515-4509-9c29-0aa3a4b6c25e.png)

**Comment by devenmatthews** [comment]
`transactionId` is still used by `get_transaction_hash_by_id` see here: 
https://alpha-mainnet.starknet.io/feeder_gateway/get_transaction_hash_by_id?transactionId=100

Although `GetTransactionStatus` `GetTransactionReceipt` and `GetTransaction` are the only ones using `TxnIdentifier` and they do not support requests by `transactionId`.

I don't think it would make sense to use `TxnIdentifier` for `get_transaction_hash_by_id`. So potentially this can be removed.

I'd like to learn more about how feeder_gateway changes- because I'm unsure why `GetTransaction` does not support `txId`. I'm wondering if support may come in the future.

**Comment by D-DePablos** [comment]
I agree that feeder_gateway allows to go from hash <-> ID, but ID is not used aside from these calls. We can keep the functionality but I will not implement tx ID as it does not work with any other calls, and add a warning when hash is changed to ID. 

The cairo-lang starknet CLI does not use any tx id as far as I have been able to see.
